config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```diff
@@ -85,39 +112,72 @@ spec:
         keyPassPath: ds/keystore.pin
         keytoolAliases:
           - name: ca-cert
-            cmd: importkeystore
+            cmd: importcert
             sourcePath: "platform-ca/ca"
-          - name: ssl-keypair
+          - name: ssl-key-pair
             cmd: importkeystore
             sourcePath: "ds/ssl-key-pair"
-          - name: master-keypair
+            isKeyPair: true
+          - name: master-key
             cmd: importkeystore
             sourcePath: "ds/master-key-pair"
-
+            isKeyPair: true
+          - name: configstorepwd
+            cmd: importpassword
+            sourcePath: am-passwords/.storepass
+            # sourcePath: "ds-passwords/tmpdsameuserpwd"
+          - name: dsameuserpwd
+            cmd: importpassword
+            sourcePath: "ds-passwords/dirmanager.pw"
+  # AM secrets
+  - name: am-env-secrets
+    keys:
+    - name: "AM_AUTHENTICATION_SHARED_SECRET"
```

Comment:
base64 encode this. See 
https://stash.forgerock.org/users/warren.strange_forgerock.com/repos/forgeops/browse/docker/7.0/am/custom-entrypoint.sh 
We need to base64 a few of the secrets